src/people/widgetViews/UserTicketsView.tsx

```diff
@@ -129,7 +129,7 @@ const UserTickets = () => {
   const listItems =
     displayedBounties && displayedBounties.length ? (
       displayedBounties.map((item: any, i: number) => {
-        const person = main.people.find((p: any) => p.owner_pubkey === item.body.owner_id);
+        const { person } = item;
```

Comment:
This change will give the assignee the ability to edit the bounty assigned to him.

Reply:
Hi @elraphty,
No, this change will not give the assignee the ability to edit the bounty assigned to him.